### PR TITLE
415: Venue page styling

### PR DESCRIPTION
**Pull request by maxb94**

### CODE REVIEW

File: src/api/admin.ts [hunk 1]
```diff
@@ -40,7 +41,8 @@ type FirestoreVenueInput = Omit<
   mapIconImageUrl?: string;
 };
 
-export const createUrlSafeName = (name: string) => name.replace(/\W/g, "");
+export const createUrlSafeName = (name: string) =>
+  `${ROOT_URL}/v/${name.replace(/\W/g, "")}`;
```
Comment (mathieud) [inline src/api/admin.ts:45]: Why do we need this?

File: src/pages/Account/Venue/TemplateForm.tsx [hunk 2]
```diff
@@ -73,6 +73,7 @@ const TemplateFormLeft: React.FC<TemplateFormLeftProps> = (props) => {
         </div>
       </div>
       <div className="page-container-left-bottombar">
+        <div />
```
Comment (mathieud) [inline src/pages/Account/Venue/TemplateForm.tsx:76]: What does this do?

File: src/api/admin.ts (hunk 1 above)
Comment (maxb94) [reply src/api/admin.ts:45]: for displaying the text 'The url of your venue will be sparkle.space/s/mylovelyvenue'. I noticed you are also using this function so will place the ROOT_URL in the caller.

File: src/pages/Account/Venue/TemplateForm.tsx (hunk 2 above)
Comment (maxb94) [reply src/pages/Account/Venue/TemplateForm.tsx:76]: 'justify-content: space-between' to ensure the button is to the right

File: src/api/admin.ts (hunk 1 above)
Comment (mathieud) [reply src/api/admin.ts:45]: That's one more config variable we have to maintain when we could read it in the browser.

File: src/pages/Account/Venue/TemplateForm.tsx (hunk 2 above)
Comment (mathieud) [reply src/pages/Account/Venue/TemplateForm.tsx:76]: Can't we have `justify-content: end`?

Comment (maxb94) [reply src/pages/Account/Venue/TemplateForm.tsx:76]: the next screen has a 'go back' button on the left hand side

File: src/api/admin.ts (hunk 1 above)
Comment (maxb94) [reply src/api/admin.ts]: hmm could do that i suppose

File: src/pages/Account/Venue/DetailsForm.tsx [hunk 3]
```diff
@@ -149,32 +155,36 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = (props) => {
     state.templatePage?.template.type === "ZOOM_ROOM" ||
     state.templatePage?.template.type === "PERFORMANCE_VENUE";
 
-  const urlSafeName = values.name ? createUrlSafeName(values.name) : undefined;
+  const urlSafeName = values.name
+    ? `/${window.location.host}/v/${createUrlSafeName(values.name)}`
```
Comment (mathieud) [inline src/pages/Account/Venue/DetailsForm.tsx:159]: Does the initial slash work here?

**Pull request merged**